node_modules/col-canvas/lib/poller.js

```diff
@@ -626,106 +626,133 @@ var handleSubmission = function(ctx, users, assignment, activities, submissionAs
     'objectType': CollabosphereConstants.ACTIVITY.OBJECT_TYPES.CANVAS_SUBMISSION,
     'metadata': {
       'submission_id': submission.id,
-      'attempt': submission.attempt
+      'attempt': submission.attempt,
+      'sync_files': assignment.category.visible
```

Comment:
Thanks for attending to the parts of speech, @johncrossman. Key has been changed to `file_sync_enabled`.